index.html

```diff
@@ -8,76 +8,40 @@
-            </li>
-        </ul>
-
-        <!-- Below is for logo construction -->
-        <div class="flex-logo-container">
-            <div class="letter-item" data-order="1">S</div> 
-            <div class="letter-item" data-order="2">q</div>
-            <div class="letter-item" data-order="3">u</div>
-            <div class="letter-item" data-order="4">u</div>
-            <div class="letter-item" data-order="5">g</div>
-            <div class="letter-item" data-order="6">l</div>
-            <div class="letter-item" data-order="7">e</div>
-        </div>
- 
-        <!-- Search form construction a-->
-            <form class="main-search-form"action="search-results.html">
-                <input class="main-search-input" type="text" name="text" 
-                class="search" placeholder="The world's greatest search engine!">
-                <button class="search-button-clickable"
-                    style="border: 0; background: transparent"> 
-                    <!-- figuring out why I can't style the above line in CSS -->
-                    
-                    <a href="search-results.html" 
-                    class="search-button-input" target="_blank">
-                    <img class="search-button"
-                    src="images/2x/outline_search_black_24dp.png" alt="">
-                    </a>
-                </button>
-            </form>
-
-        <!-- Footer bar construction -->
-        <footer class="main-footer-bar">
-            <ul class="footer-bar">
-                <li class="footer-bar-item">
-                    <a id="footer-link" href="search-results.html">About</a>
-                </li>
-                <li class="footer-bar-item">
-                    <a id="footer-link" href="search-results.html">Terms</a>
-                </li>
-                <li class="footer-bar-item">
-                    <a id="footer-link" href="search-results.html">Settings</a>
-                </li>
-
-            </ul>
-        </footer>
+        <nav class="main-nav-bar"> <!-- Top Navbar -->
+            <div class="menu">
+                <div class="main-nav-item" href="#">SqMail</a></div>
+                <div class="main-nav-item" href="#">Images</a></div>
+                <div class="main-nav-item" href="#">Apps</a></div>
+                <div class="main-nav-item" href="#">Account</a></div>
+            </div>
+        </nav>
+        <main>
+                <section class="main-content"> <!-- Body area -->
+                    <div class="main-content-item">
+                        <div class="main-logo">
+                        Squugle
+                        </div>
+                    </div>
+                    <div class="main-content-item">
+                        <input class="main-search-bar" type="text" placeholder="Search..">
+                    </div>
+                </section>
+        </main>
+        <footer class="main-footer"> 
+                <div class="menu-footer">
```

Comment:
Double indention, btw if you re using tab indention, I recommend rather 4 spaces (you can tell IDE to put 4 spaces instead of `\t` char on tab key stroke.

FYI webstorm has it all, also analysis tools to format html code automatically

Reply:
Fixed